scripts/docs/bundle.js

```diff
@@ -0,0 +1,223 @@
+/**
+ * Copyright 2014 Google Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+var path = require('path');
+var globby = require('globby');
+var semver = require('semver');
+var mkdirp = require('mkdirp');
+var fs = require('fs');
+var async = require('async');
+var flatten = require('lodash.flatten');
+var parser = require('./parser');
+var config = require('./config');
+
+// since we version the JSON in ghpages and we don't create tags for
```

Comment:
I think it's because `npm version v0.1.0` will make a tag called `0.1.0`, which is going to clash with bigquery's 0.1.0, and storage's 0.1.0, etc. But what we will do instead is create a custom tag and tell `npm version` not to auto-create the `0.1.0` tag.